modules/ROOT/pages/external-org-access.adoc

```diff
@@ -0,0 +1,41 @@
+= External Organization Access
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Organizations that have their own, separate domains in Anypoint Platform can share resources with one another using external access lists of trusted organizations.
+This enables your organization to maintain an independent Anypoint Platform organization without keeping resources isolated from external organizations that have partnerships or shared projects. For example, if your organization and another organization add each other to their external access lists, both organizations can share resources with each other through Anypoint Exchange.
+
+As an organization administrator, you can specify which external organizations can share resources with your organization.
+In turn, an external organization that has your organization's unique domain name can add your organization to its list of trusted organizations so that your organization can share assets with it. Both organizations must have one another on their external access lists to share resources.
```

Comment:
```suggestion
In turn, an external organization that has your organization's unique domain name can add your organization to its external access list so that your organization can share assets with it. Both organizations must have one another on their external access lists to share resources.
```